packages/pyright-internal/src/common/pathUtils.ts

```diff
@@ -536,7 +536,7 @@ export function stripFileExtension(fileName: string, multiDotExtension = false)
 }
 
 export function realCasePath(pathString: string, fileSystem: ReadOnlyFileSystem): string {
-    return normalizePath(fileSystem.realCasePath(pathString));
```

Comment:
This normalizePath can change the empty string into a `.`. We shouldn't need to normalize since calling real case o n a path will have that same effect anyway.